### ISSUE TITLE
Hashtag feature request in posts and search

### ISSUE THREAD

**Issue by mrgodgiven** [WWBN/AVideo#6718]
Greetings @DanielnetoDotCom  I would like to request a feature of hashtags on search and posts, because it helps users to get trending videos of the same hashtag and a searchable hashtag is a very important feature, please add 🙏
![Screenshot_20220608_044552](https://user-images.githubusercontent.com/72950503/172521910-f9168d0f-e3a9-45f6-8770-d8c9d6383188.jpg)

**Comment by walt93** [comment]
Daniel, I agree here, and as a recommendation, this isn't conceptually different from tags. 

In the Share menu when you share to Twitter, Facebook, etc... In addition to the video title and URL, you would put the tags - with the hashmark so #tag1 #tag2 #tag3 - and then those are used in the social media site.

You would also do the same for auto scheduled posts to Twitter.

This is a fantastic idea, it puts out self-reinforcing ways of growing site traffic. 💡😎

**Comment by DanielnetoDotCom** [comment]
Did you tried that

https://github.com/WWBN/AVideo/wiki/VideoTags-Plugin

**Comment by mrgodgiven** [comment]
Yes I did but it's not the same as hashtags , hashtags are much different
and better

**Comment by DanielnetoDotCom** [comment]
what is the difference?

**Comment by mrgodgiven** [comment]
Please take a look at this
https://www.meetmycraft.live/hashtag/ggpradioairplay
![Screenshot (1)](https://user-images.githubusercontent.com/72950503/172710467-c1a73dca-6812-4ed0-9fe1-9f4019442549.png)

**Comment by DanielnetoDotCom** [comment]
those are hashtags on comments. at the moment we do not have that.

**Comment by mrgodgiven** [comment]
I would be gratefull if we could have it in the future, Because it would take the platform to another level. I noticed about 1500 users trying to use it in the platform it's not there which is the reason why I decided to request it. Avideo has become a great plaform and those hashtags would make it even greater
